bin/plugin/commands/common.js

```diff
@@ -133,3 +96,52 @@ exports.getModuleData = async ( modulesDir ) => {
 			return firstModule.slug.localeCompare( secondModule.slug );
 		} );
 };
+
+/**
+ * Returns the list of module data for all modules.
+ *
+ * @param {string} moduleFile Modules file.
+ *
+ * @return {[]WPModuleData} Module data list.
+ */
+exports.getModuleDataFromHeaders = ( moduleFile ) => {
+	// Populate slug and focus based on file path.
+	const moduleDir = path.dirname( moduleFile );
+	const moduleData = {
+		slug: path.basename( moduleDir ),
+		focus: path.basename( path.dirname( moduleDir ) ),
+	};
```

Comment:
This logic should rather be kept in the original `getModuleData()` function, since this is **not** information that it gets from the module header. Let's keep those concerns separated, so that each function only has a single purpose.